### DISCUSSION TITLE
Batch size for 80GB GPU memory

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#6726] [Q&A] [answered]
In [here](https://github.com/NVIDIA/NeMo/blob/0150b910de0623b4082af35171ad8d6406379c4e/examples/asr/conf/conformer/conformer_ctc_bpe.yaml#L27) you are suggesting batch 64 for 80 GB GPU memory. Which model type (small, medium, large, xlarge) does this batch size applicable?

**Comment by titu1994** [comment, marked as answer]
This is for the Large model (the default in the config) with 120 M params

**Reply by mehadi92** [reply]
@titu1994 ,

you are suggesting batch size 32 for 32G GPU memory. But for 80G GPU memory, you are suggesting batch size 64. Is there any specific reason for it? should not it be 80 batch size for GPU memory 80G? 

Thanks

**Reply by titu1994** [reply]
Training memory doesn't work like that. It's not 1 gb per sample.
